misk-slack/src/main/kotlin/misk/slack/webapi/FakeSlackApi.kt

```diff
@@ -0,0 +1,22 @@
+package misk.slack.webapi
+
+import misk.slack.webapi.helpers.PostMessage
+import misk.slack.webapi.helpers.PostMessageResponse
+import retrofit2.Call
+import retrofit2.mock.Calls
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class FakeSlackApi @Inject constructor() : SlackApi {
```

Comment:
I don't think a Fake / Real is necessary for SlackApi. I haven't seen that before and am not sure where it would be necessary. Fake / Real is more useful at the client layer abstraction.

Reply:
Originally, I paired this with the `FakeClient`, but the functionality for the fake didn't diverge from the `RealClient`, so I deleted it. The `PingSlackActionTest` also supports testing, so I think we can remove this.